apps/frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,12 @@
+import { iconNameToSvg } from "~/libs/maps/maps.js";
+import { IconName } from "~/libs/types/icon-name.type.js";
+
+type Properties = {
+	name: IconName;
+};
+
+const Icon: React.FC<Properties> = ({ name }: Properties) => {
```

Comment:
I think you should add an alt prop and a prop like "variant" that will be used for the class name

Reply:
I kinda agree on the alt prop, don't know about variant. If I understand you correctly, the usage sould be something like ` variant={styles[sidebarIcon]}`? @fshabanov  what do you think?

Reply:
Somethink like in Link component with prop "type"